src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,9 +4,73 @@
   pageTitle="Tabs"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Tabs are containers for dividing large amounts of content on a page.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include counts"
+          propertyName="includeCounts"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include add and close"
+          propertyName="includeAddAndClose"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-tabset *ngIf="!demoSettings.includeAddAndClose">
+      <sky-tab *ngFor="let tab of tabArray; let i = index"
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+
+    <sky-tabset *ngIf="demoSettings.includeAddAndClose"
+      (newTab)="onNewTabClick()"
+    >
+      <sky-tab *ngFor="let tab of tabArray; let i = index"
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+        (close)="onCloseClick(i)"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-code-examples>
+    <sky-docs-code-example
+      heading="Regular tabs"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/regular"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs bound to an array"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Tabs with counts"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/counts"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Tabs with add and close"
```

Comment:
Does it make sense to demonstrate the add and close functionality when the tabs are static? I considered this but in the end set up this particular demo to use a dynamic tabset bound to an array. It just seemed to be more of a realistic scenario that way. Should we just combine "Dynamic tabs" and "add and close" into one code example?

Reply:
Might be a question for @Blackbaud-AdamFunderburk.

Reply:
It depends on the practical interaction between the two. Can you have "Add and close" behavior on tabs _without_ using dynamic tabs bound to an array? Conversely, would you ever use an array for dynamic tabs when they _didn't_ use "Add and close"?

If the only practical reason to do one is to get the other, and you'd never separate them, then combining them into a single example is definitely the right way to go.

Reply:
Yeah, you could totally have an add/close for static tabs - there's nothing stopping you. So would you like to see add/close examples for both dynamic and static?

Reply:
It sounds like these things are true, but correct me if they're not:
1 - Deciding whether or not to use add/close is a _design decision_, based on the interaction scenario.
2 - Deciding whether to use static or dynamic tabs has _nothing_ to do with the interaction design, but instead depends on an _implementation concern_ of what information is creating the tabset.
3 - The implementation of the add/close behavior is different enough between static and dynamic tabs that developers would benefit from an example of each.

If those are true, then I think you're right and separate examples are best. That raises some another small question:
Is it right to describe the opposite of dynamic tabs as "Regular tabs", or should we call them "Static tabs"?
That's just a matter of labeling the examples, though.
![image](https://user-images.githubusercontent.com/15896944/71836086-408b3200-3081-11ea-835c-930f4948b612.png)

Reply:
Yep, that all checks out. All your assumptions are correct.

"static" would be much better. Regular doesn't really have any inherit meaning, IMHO.

Reply:
The only argument for labeling one of them "regular" would be if there were lots of cases where either implementation _could_ work, but we strongly recommend one of them. If that's not the case, then static vs dynamic is good.

Thanks for walking through it with me. I think we've got the right answer now.